lib/cloudregister/registerutils.py

```diff
@@ -1257,3 +1267,67 @@ def __replace_url_target(config_files, new_smt):
                 current_service_server,
                 new_smt.get_FQDN()))
             new_config.close()
+
+
+# ----------------------------------------------------------------------------
+def _get_license_type():
```

Comment:
But the server makes this decision, why do we have to care about he license type on the client side as well?